recipe-box-backend/src/main/java/com/endava/recipebox/controller/RecipeController.java

```diff
@@ -35,5 +38,14 @@ public ResponseEntity<List<RecipeDTO>> getAllPublicRecipesByName(@RequestParam S
         return ResponseEntity.ok(recipeService.getAllPublicRecipesByName(recipeName));
     }
 
+    @PostMapping("/create")
+    public ResponseEntity<Recipe> createRecipe(@RequestBody RecipeRequestDTO recipeRequestDTO, @RequestParam Long userId) {
+        if (recipeRequestDTO != null && userId != null) {
+            Recipe createdRecipe = recipeService.createRecipe(recipeRequestDTO, userId);
+            return ResponseEntity.ok(createdRecipe);
```

Comment:
On the response is enough to send the confirmation that the recipe was created successfully. There is no need to return the request body back. You can send on response "Recipe added successfully."